apps/web/pages/api/trpc/slots/[trpc].ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * This file contains tRPC's HTTP response handler
+ */
+import { z } from "zod";
+
+import { getServerSession } from "@calcom/features/auth/lib/getServerSession";
```

Comment:
We can also test if it helps to remove this import. It's quite big from Lukas's findings and the slots router is all publicProcedures, so just a `() => null` would work as a session getter for this endpoint/router.